sdks/go/test/integration/internal/jars/run_nonunix.go

```diff
@@ -29,7 +30,7 @@ import (
 // non-unix version does not handle timeout durations.
 func getTimeoutRunner() runCallback {
 	// Wrap run with error handling for OS that does not support timeout duration.
-	return func(dur time.Duration, jar string, args ...string) (*Process, error) {
+	return func(dur time.Duration, jar string, args ...string) (Process, error) {
```

Comment:
Curious if this change was necessary?

Reply:
This is one of the build breaks. The contract has `Process` rather than `*Process` because it's an interface.